roboprop_client/export_model.py

```diff
@@ -0,0 +1,101 @@
+import bpy
+import os
+from xml.dom import minidom
+from xml.etree import ElementTree
+from pathlib import Path
+
+
+def export_sdf(path_model: Path, model_name: str):
+    path_collision = os.path.join(path_model, "assets/collision.obj")
```

Comment:
I don't see this variable being used. How is this supposed to work?
I am quite curious wether:
* blender kit provides collision meshes
* if not, do we copy the visual mesh? if so we need to be careful as we may have to decimate the mesh to allow for fast enough physics, but this is a bit case by case

Reply:
Collision meshes is brought in by Andras' PR here: https://github.com/art-e-fact/RoboProp/pull/37/files so should be functional after that -> is that right @azazdeaz ?

Reply:
Generating collision meshes are still on the TODO list. I just created an issue for this: https://github.com/art-e-fact/RoboProp/issues/42. We could just use the visual mesh, but as Fabian mentioned, it would be better to generate a simplified collisions mesh. Luckily, Blender has a bunch of tools for this, so it should be easy to implement.

Reply:
for starters, it is ok to start withe the visual mesh